pkg/deploy/config.go

```diff
@@ -147,6 +150,13 @@ func (conf *RPConfig) validate() error {
 	v := reflect.ValueOf(*configuration)
 	missingFields := []string{}
 
+	if conf.Configuration.SSHPublicKey == nil {
+		key, _ := rsa.GenerateKey(rand.Reader, 2048)
+		pubKey := x509.MarshalPKCS1PublicKey(&key.PublicKey)
+		publicKey := string(pubKey)
+		conf.Configuration.SSHPublicKey = (&publicKey)
+	}
```

Comment:
If I'm understanding the description correctly we no longer need a permanently-stored key for SSH access to the servers, which means we don't care that the generated private key is lost after this block?

Reply:
Correct! We actually want to forget about it and have garbage collection kick in.